src/field/field255/integer.rs

```diff
@@ -0,0 +1,190 @@
+// Copyright (c) 2023 ISRG
```

Comment:
I don't understand the purpose of this module. Is it so we can compare the performance of a bigint based `Field255` against the `fiat_crypto` one?

Reply:
No, this primarily exists to provide a big-enough integer type that we can run `field_element_test_common()` on `Field255`. The `Field255Integer` type and its impls here are needed to satisfy the `Integer` associated type on the relevant `TestFieldElementWithInteger` trait.

Reply:
Suggestion: The amount of code in this module is now small enough that I think it would be reaonable to fold into the `tests` module in field255.rs